frontend/src/pages/home/resources/useResourcesSection.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { useNavigate } from 'react-router-dom';
+import {
+  Button,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateHeader,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  PageSection,
+  Stack,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { OdhDocument } from '~/types';
+import OdhDocCard from '~/components/OdhDocCard';
+import { useDocResources } from '~/pages/learningCenter/useDocResources';
+import ScrolledGallery from '~/concepts/design/ScrolledGallery';
+import CollapsibleSection from '~/concepts/design/CollapsibleSection';
+import { useBrowserStorage } from '~/components/browserStorage';
+import ResourcesLoading from './ResourcesLoading';
+
+const includedCards = [
+  { name: 'create-jupyter-notebook', kind: 'OdhQuickStart' },
+  { name: 'rhoai-tutorial-fraud', kind: 'OdhDocument' },
+  { name: 'rhoai-documentation', kind: 'OdhDocument' },
+  { name: 'watson-x-use-case', kind: 'OdhDocument' },
+  { name: 'openvino-inference-notebook', kind: 'OdhQuickStart' },
+];
+
+export const useResourcesSection = (): React.ReactNode => {
+  const navigate = useNavigate();
+  const [shownResources, setShownResources] = React.useState<OdhDocument[]>([]);
+  const { docs, loaded, loadError } = useDocResources();
+  const [resourcesOpen, setResourcesOpen] = useBrowserStorage<boolean>(
+    'odh.home.resources.open',
+    true,
+    true,
+    false,
+  );
+
+  React.useEffect(() => {
+    if (loaded && !loadError) {
+      setShownResources(
+        includedCards.reduce((acc, included) => {
+          const doc = docs.find(
+            (d) => d.metadata.name === included.name && d.kind === included.kind,
+          );
+          if (doc) {
+            acc.push(doc);
+          }
+          return acc;
+        }, [] as OdhDocument[]),
+      );
+    }
+  }, [loaded, loadError, docs]);
```

Comment:
Two comments
1. This effect is a build step, it ties your state & useDoc together -- this screams custom hook and isolate the logic away from the component
2. Avoid `as` -- `.reduce` is a Generic function, use the Generics